tests/constraints.txt

```diff
@@ -0,0 +1,6 @@
+importlib-metadata==0.22;python_version < "3.8"
+packaging==19.0
+pep517==0.9.1
+toml==0.10.0
+typing==3.5.2.2;python_version < "3"
```

Comment:
Hmm, why is the minimum version here different from the minimum version in `setup.cfg`?

Reply:
If you are referring to typing==3.5.2.2, there wasn't a `typing==3.5.2` release (took several tries apparently to get it on PyPI), so this is the actual first version available for download. If you really think it should be written out in the minimums, it can be, but a four digit minimum seems excessive, and there is no 3.5.2 or 3.5.2.1 to cause an issue here. Can change if you prefer, though.

If it's another one, then that's likely a mistake.

Reply:
It looks like the uploaded sdist for 3.5.2 has a version number of 3.5.2.2, which is the version pip is seeing on the simple API.  Ah well, as long as it works and there's no intermediate versions.

Reply:
From the tag:

> ```
> Bump version to 3.5.2.2.
> (That's the first one I successfully uploaded to PyPI.)
> ```